src/components/common/modal/folderNameInput/folderNameInput.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
+import styles from './FolderNameInput.module.css';
+
+const FolderNameInput = ({ folderName = '' }) => {
+  const [value, setValue] = useState(folderName);
+  const handleChange = e => setValue(e.target.value);
+  const active = value ? 'active' : '';
+
+  return (
+    <input
+      type="text"
+      placeholder="내용 입력"
+      value={value}
+      className={`${styles.input} ${active && styles.active}`}

```

Comment:
위에서 active 변수 선언하지 않고 바로 넣어도 될 거 같아요!
```suggestion
      className={`${styles.input} ${value && styles.active}`}

```